drivers/dac/ad5758/ad5758.c

```diff
@@ -310,9 +311,11 @@ int32_t ad5758_set_dc_dc_conv_mode(struct ad5758_dev *dev,
 		ad5758_spi_reg_read(dev, AD5758_REG_DCDC_CONFIG2, &reg_data);
 	} while (reg_data & AD5758_DCDC_CONFIG2_BUSY_3WI_MSK);
 
-	dev->dc_dc_mode = mode;
```

Comment:
Do we not want to save the new mode in the handler anymore? If we don't we could erase the dc_dc_mode member from the struct.